compiler/optimizations/loopInvariantCodeMotion.cpp

```diff
@@ -742,7 +742,7 @@ static void computeLoopInvariants(std::vector<SymExpr*>& loopInvariants,
     if(numAliases > MAX_NUM_ALIASES) {
 #ifdef detailedTiming
        FILE* tooManyAliasesFile = fopen(astr(CHPL_HOME,"/LICMaliases.txt"), "a");
-       fprintf(tooManyAliasesFile, "Skipping fn %s %d of module %s %d "
+       fprintf(tooManyAliasesFile, "Skipping fnsd of modulesd "
```

Comment:
Looks like all the `%` formatting got messed up. For removing trailing whitespace in vim I think I usually do `:%s/\s\+$//e`, but I'm not sure if you cleaned up more than just trailing whitespace